src/tools/css/get-css.js

```diff
@@ -50,11 +50,13 @@ const getAvailableFeatures = (font) => {
 };
 
 const getVariableCSS = (font) => {
```

Comment:
Isn't this covered by tests?

Reply:
Not specifically this call, but the result of it is checked in https://github.com/Wakamai-Fondue/wakamai-fondue-engine/blob/master/test/Fondue.test.js#L52